i18n/config.ts

```diff
@@ -0,0 +1,29 @@
+import i18n from 'i18next';
+import Backend from 'i18next-xhr-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+
+i18n
+  // load translation using xhr -> see /i18n/translations
+  // learn more: https://github.com/i18next/i18next-xhr-backend
+  .use(Backend)
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
```

Comment:
One reminder, the language must be changed vi a configuration rather via browser detection, you can check the description here https://github.com/verdaccio/ui/issues/102

Reply:
So you need:
- [ ] Add new option https://github.com/verdaccio/verdaccio/blob/master/src/api/web/index.ts#L86
- [ ] Update documentation at website how to use this new param https://github.com/verdaccio/website/blob/master/docs/web.md
- [ ] The language must be taken via configuration and the UI render it

Reply:
I am preparing a PR to simplifying the config passed to verdaccio to UI, I will include language so you can use real conf.